olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml

```diff
@@ -120,22 +120,20 @@ spec:
 
     - Rapid deployment of serverless containers
     - Automatic scaling up and down to zero
-    - Routing and network programming for Istio components
+    - Routing and network programming
     - Point-in-time snapshots of deployed code and configurations
 
     ## Prerequisites
 
     The Serverless Operator's provided APIs such as Knative Serving
     have certain requirements with regards to the size of the underlying
-    cluster and a working installation of Service Mesh. See the [installation
-    section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html-single/serverless/index#installing-openshift-serverless)
-    of the Serverless documentation for more info.
+    cluster. See [Getting started with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index#serverless-getting-started)
+    for more info.
 
     ## Further Information
 
     For documentation on using Knative Serving, see the
-    [serving section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html-single/serverless/index#knative-serving_serverless-architecture) of the
-    [Serverless documentation site](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html-single/serverless/index).
+    [serving section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index#knative-serving).
```

Comment:
Why change the links here and remove the Serverless documentation site link?

Reply:
Per slack discussion, some doc sections were moved around.